exercises/bank-account/bank-account.js

```diff
@@ -0,0 +1,34 @@
+//
+// This is only a SKELETON file for the 'Bank Account' exercise. It's been provided as a
+// convenience to get you started writing code faster.
+//
+
+export class BankAccount {
+
+  constructor() {
+    throw new Error('Remove this statement and implement this function');
+  }
+
+  open() {
+    throw new Error('Remove this statement and implement this function');
+  }
+
+  close() {
+    throw new Error('Remove this statement and implement this function');
+  }
+
+  deposit() {
+    throw new Error('Remove this statement and implement this function');
+  }
+
+  withdraw() {
+    throw new Error('Remove this statement and implement this function');
+  }
+
+  get balance() {
+    throw new Error('Remove this statement and implement this function');
+  }
+
+}
+
+export class ValueError extends Error {}
```

Comment:
Can you add a constructor that sends to `super('A descriptive error message')` so the user better understands what a `ValueError` is?